op-challenger/game/types/types.go

```diff
@@ -68,7 +69,21 @@ func (m LargePreimageMetaData) ShouldVerify() bool {
 	return m.Timestamp > 0 && !m.Countered
 }
 
+const LeafSize = 136
```

Comment:
Ah nice! Since this is hoisted to the higher level types package, I think we can also reference this from the `keccak.matrix` package [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-challenger/game/keccak/matrix/matrix.go#L18-L19)